include/internal/libspdm_lib_config.h

```diff
@@ -107,22 +107,65 @@
 
 /*when in FIPS mode, only support approved algo in FIPS */
 #if LIBSPDM_FIPS_MODE
-#undef LIBSPDM_SM2_DSA_SUPPORT
-#define LIBSPDM_SM2_DSA_SUPPORT 0
+#undef LIBSPDM_SM2_DSA_P256_SUPPORT
+#define LIBSPDM_SM2_DSA_P256_SUPPORT 0
 
-#undef LIBSPDM_SM2_KEY_EXCHANGE_SUPPORT
-#define LIBSPDM_SM2_KEY_EXCHANGE_SUPPORT 0
+#undef LIBSPDM_SM2_KEY_EXCHANGE_P256_SUPPORT
+#define LIBSPDM_SM2_KEY_EXCHANGE_P256_SUPPORT 0
 
 #undef LIBSPDM_AEAD_CHACHA20_POLY1305_SUPPORT
 #define LIBSPDM_AEAD_CHACHA20_POLY1305_SUPPORT 0
 
-#undef LIBSPDM_AEAD_SM4_SUPPORT
-#define LIBSPDM_AEAD_SM4_SUPPORT 0
+#undef LIBSPDM_AEAD_SM4_128_GCM_SUPPORT
+#define LIBSPDM_AEAD_SM4_128_GCM_SUPPORT 0
 
 #undef LIBSPDM_SM3_256_SUPPORT
 #define LIBSPDM_SM3_256_SUPPORT 0
 #endif /*LIBSPDM_FIPS_MODE*/
 
+/* define crypto algorithm without parameter */
+#define LIBSPDM_RSA_SSA_SUPPORT (LIBSPDM_RSA_SSA_2048_SUPPORT || \
```

Comment:
All the macros used in expressions need `()` around them.

Reply:
Added.